draft-ietf-tls-rfc8446bis.md

```diff
@@ -2304,6 +2304,16 @@ MAY omit the most preferred groups. Such a situation could arise if the most
 preferred groups are new and unlikely to be supported in enough places to
 make pregenerating key shares for them efficient.
 
+For this reason, the omission of a share for group A and inclusion of
+one for group B does not mean that the client prefers B to A.
+Selecting a group based on KeyShareEntry may result in the use of
+a less preferred group than the client and server mutually support,
+though saving the round trip of HelloRetryRequest. Servers
+that wish to respect the client's group preferences SHOULD first
```

Comment:
Nit: This may not just be client prefs, but also server prefs. I think really this is about the server not believing in equipreference groups, either by way of the client's prefs (since the client can't express them) or its own.

But talking about equipreference groups in a spec that intentionally doesn't talk about selection criteria is kinda weird. Not sure what to do here. Maybe we don't need that example if we've spelled it out in the preceding text clearly enough?

Reply:
That's my hope.